examples/Example3_ChangeAddress/Example3_ChangeAddress.ino

```diff
@@ -0,0 +1,71 @@
+/* SparkFun Ulrasonic Distance Sensor - Example 3 Changing the Ultrasonic's Address
+ * To reset the original I2C address, ground the "RST" pad on the backside of the board.
+ * by touching a wire to the pad and then to ground. The address will be reset to 0x2F.
+ * 
+ * Product: 
+ *  *  SparkFun Qwiic Ultrasonic Distance Sensor - HC-SR04 (SEN-1XXXX)
+ *  *  https://www.sparkfun.com/1XXXX
+ * 
+ * Written By: Elias Santistevan
+ * Date: 06/2024
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ * Copyright (c) 2024 SparkFun Electronics
+ */
+
+#include "SparkFun_Qwiic_Ultrasonic_Arduino_Library.h"
+
+// Create an ultrasonic sensor object
+QwiicUltrasonic myUltrasonic;
+
+// Here we set the device address. Note that an older version of the Qwiic
+// Ultrasonic firmware used a default address of 0x00. If yours uses 0x00,
+// you'll need to change the address below. It is also recommended to run
+// Example 2 to change the address to the new default.
+uint8_t deviceAddress = kQwiicUltrasonicDefaultAddress; // 0x2F
+// uint8_t deviceAddress = 0x00;
+
+// New addres is 7-bit unshifted.
+uint8_t NEW_ADDR = 0x1E;
+
+void setup()
+{
+    // Start serial
+  Serial.begin(115200);
+  Serial.println("Ultrasonic Distance Sensor - Example 4 - Change Address");
+
+  Wire.begin();
+
+  // This sketch wont' run until you open the serial monitor
+  while(!Serial)
+    ; 
+
+  // Attempt to begin the sensor
+  if (myUltrasonic.begin(kQwiicUltrasonicDefaultAddress) == false )
+  {
+    Serial.println("Ultrasonic sensor not connected, check your wiring and I2C address!");
+  }
+
+  Serial.println("Ready to change address.");
+  delay(1000);
+
+  Serial.print("Changing Address To: ");
+  Serial.println(NEW_ADDR, HEX);
+
+  // Call change address.....
+  myUltrasonic.changeAddress(NEW_ADDR);
+  delay(1000);
+  
+
+  Serial.print("Load up example 1 with the new address at: ");
+  Serial.println(NEW_ADDR, HEX);
+  Serial.println("Freezing....");
+  while(1)
+    ;
+
+}
+
+void loop()
+{
+}
```

Comment:
Would prefer for this example to continue like Example 1, rather than directing users to load another example, since that adds a couple potential points of failure.